### DISCUSSION TITLE
Blender animations in hubs

### DISCUSSION THREAD

**Discussion by kamijoan** [mozilla/hubs#2894] [General Questions & Discussion] [answered]
Can someone help me? I'm exporting a scene from Blender and loading it on Spoke, it contains multiple animated objects and they are all on the list in "loop animation" but I can only choose one. Can I group them somehow on blender so they all reproduce?

**Comment by kamijoan** [comment]
Ended up exporting it as .fbx over to cinema4d, the .glb from cinema bakes every animation and spoke reads it on one loop, still, you need cinema4d for that. Still open for answers in blender.

**Comment by j-conrad** [comment, marked as answer]
https://docs.blender.org/manual/en/latest/addons/import_export/scene_gltf2.html#animation)

Each object needs to have its animation 'pushed' or 'stashed' into an NLA track--all with the same name. Since the default .glb export setting 'Group by NLA Track' is ON by default, this will ensure that any objects that have NLA tracks with the same name get exported as one animation.
Then, in Spoke, you'll only see one item in the animation dropdown list and all items will animate together.

**Reply by kamijoan** [reply]
Tested it and actually it ended up being the opposite.
![image](https://user-images.githubusercontent.com/49537119/90841944-aa5fff80-e323-11ea-9781-ef406eedb978.png)
If no animation is stashed and that option is disabled it works perfectly. Most of the issue was that I wasn't on the latest version of blender and that toggle wasn't available.
